src/modules/statusUpdate.js

```diff
@@ -0,0 +1,25 @@
+export const updateStatus = (index) => {
+  const check = document.getElementById(`check${index}`);
+  const description = document.getElementById(`description${index}`);
+  const toDoList = JSON.parse(localStorage.getItem('allEntries'));
+  if (check.checked) {
+    toDoList[index].completed = true;
+    description.style.textDecoration = 'line-through';
+  } else {
+    toDoList[index].completed = false;
+    description.style.textDecoration = 'none';
+  }
```

Comment:
- [ ] Great work implementing this feature this way. However, you are meant to change the status of tasks by adding a `change` `eventlistener` to the checkbox. Kindly implement this feature. 👍 

![image](https://user-images.githubusercontent.com/77106962/206593814-ad36be6b-cec1-407d-bbe2-3a46802716c4.png)

`checkbox.addEventlistener('change', callbackFunction)`